lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -81,7 +81,8 @@ UsdPrim ufePathToPrim(const Ufe::Path& path)
 
     UsdPrim prim;
     if (auto stage = getStage(Ufe::Path(segments[0]))) {
-        prim = stage->GetPrimAtPath(SdfPath(segments[1].string()));
+        const SdfPath usdPath = SdfPath(segments[1].string());
+        prim = stage->GetPrimAtPath(usdPath.GetPrimPath());
```

Comment:
So maybe I lied... `SdfPath` here is getting a string that will include the `instance_XXX` that we added to the `Ufe::Path` in the render delegate, so we may actually be polluting things on the USD side as well.

In any case, this ensures that if we did tack on the instance component of the path there, we strip it back off when getting the `UsdPrim` for that instance.